src/util/fields.py

```diff
@@ -202,6 +322,9 @@ def clean_context(context):
             [(fieldname, fieldname + " desc"), model, r"\y{}\y".format(fieldname)],
         )
 
+    # ir.exports
+    remove_ir_exports_lines(cr, fields=[(model, fieldname)])
```

Comment:
Logging the number of export lines removed is not useful, we can actually log nothing.
```suggestion
    matching_exports = _get_resolved_ir_exports(cr, fields=[(model, fieldname)])
    if matching_exports:  
         cr.execute("DELETE FROM ir_exports_line WHERE id IN %s", tuple(matching_exports.keys())])
```